### ISSUE TITLE
There is too much Ammo in interloper, 

### ISSUE THREAD

**Issue by onebunchmugen** [DigitalzombieTLD/CanneryManufacturingDLC#4]
Scarcity should be the key element of Interloper but I found tons of ammo box and ingredients in pleasant valley in day one. Wth these amount of  bullets you can holocaust entire great bear island

**Comment by NakiriFox** [comment]
Agree, I collected 160 rounds of ammunition on the first day, which upset the balance of the game. Hostile creatures pose no threat at all.
